src/gitProjectManager.ts

```diff
@@ -297,7 +297,7 @@ export default class GitProjectManager {
             };
         })).then(selected => {
             if (selected) {
-                self.openProject(selected.description);
+                self.openProject(selected.description, openInNewWindow);
```

Comment:
I need to pass openInNewWindow boolean value to open  new window, if user did setting.